targets/test_utils.h

```diff
@@ -18,8 +18,8 @@
  *
```

Comment:
This should be in a separate directory, not directly under `targets` (see my earlier comment)